<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>891</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="source">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>始发地</string>
     </property>
    </widget>
    <widget class="QComboBox" name="source_box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>北京</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>上海</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>广州</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>杭州</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>沈阳</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>哈尔滨</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>成都</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>南京</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西安</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>武汉</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="travel_s">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>旅行策略</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="btntravel_s1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>风险最小</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="btntrave_s2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>60</y>
       <width>115</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>限时风险最小</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="btntravel_s3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>191</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>风险最小（含交通工具）</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="btntravel_s4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>限时风险最小（含交通工具）</string>
     </property>
    </widget>
    <widget class="QLabel" name="destination">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>目的地</string>
     </property>
    </widget>
    <widget class="QComboBox" name="destination_box">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>87</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>北京</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>上海</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>广州</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>杭州</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>沈阳</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>哈尔滨</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>成都</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>南京</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>西安</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>武汉</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_string1">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>  限时策略的时间限制</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_label">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>30</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>时间/h</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="time_input">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="new_traveller">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>50</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>新建旅客</string>
     </property>
    </widget>
    <widget class="QPushButton" name="new_tr_isok">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>60</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="new_tr_iscan">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>60</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>191</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="check_all">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>0</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>查询当前所有旅客</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="check_sb2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="check_sb3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>位旅客</string>
     </property>
    </widget>
    <widget class="QPushButton" name="check_sb">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>查询</string>
     </property>
    </widget>
    <widget class="QLabel" name="check_sb1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>第</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>暂停</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>77</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_log">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>输出日志</string>
     </property>
    </widget>
    <widget class="QPushButton" name="go_on">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>继续</string>
     </property>
    </widget>
    <widget class="QLabel" name="beisu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>倍速</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="x1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>1x</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="x2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2x</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="x5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>41</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>5x</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="x10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10x</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>260</y>
      <width>191</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="now_time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>当前时间</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_year">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>年</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_month">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>16</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>月</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_day">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>日</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_hour">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>时</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_time2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>当前时刻</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_traveller_num">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前旅客数</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_traveller2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>个旅客在旅行</string>
     </property>
    </widget>
    <widget class="QLabel" name="now_traveller1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前有</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="year">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="month">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="day">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="hour">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="travel_num">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>430</y>
      <width>191</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>881</width>
      <height>591</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="map">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>881</width>
       <height>591</height>
      </rect>
     </property>
     <property name="text">
      <string>map</string>
     </property>
    </widget>
    <widget class="QLabel" name="heb_icon">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>130</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="sy_icon">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>215</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="bj_icon">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="xa_icon">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>350</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="cd_icon">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>420</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="gz_icon">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>530</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="wh_icon">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>420</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="nj_icon">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>370</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="sh_icon">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>400</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="hz_icon">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>460</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>460</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>成都</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>390</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>西安</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>460</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>武汉</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>570</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>   广州</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>170</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>哈尔滨</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>沈阳</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>410</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>南京</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>430</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>上海</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>450</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>杭州</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
